examples/python/model-chat.py

```diff
@@ -87,6 +87,8 @@ def main(args):
     system_tokens = tokenizer.encode(system_prompt)
     generator.append_tokens(system_tokens)
     system_prompt_length = len(system_tokens)
+    ngram_len_threshold = 10
+    repeat_threshold = 5
```

Comment:
Are these values standard or should they be adjustable? If possible, a comment explaining why these values were chosen.